### DISCUSSION TITLE
Support oEmbed protocol please.

### DISCUSSION THREAD

**Discussion by fishfree** [mediacms-io/mediacms#621] [Feature Requests]
https://oembed.com/

**Comment by codeJo5h** [comment]
I am very interested in having this implemented on corelabrat.com @mgogoulos If you can provide some pointers I am using Docker and I know some modifications don't take effect or what. I'm happy to jump in and get this done just some file structure info. Am available on Skype also! Can probably get this done pretty quickly. Thank you for the awesome app.

**Comment by oderyn** [comment]
Bumping this. It would definitely be useful as we are not always able to use the embed code to insert videos.
